### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by petrparolek**
- fixed composer in CI
- upgrade PHPStan
- fixed code reported by PHPStan
- CS fix

### CODE REVIEW

File: src/ImageStorage.php [hunk 1]
```diff
@@ -124,7 +124,7 @@ public function saveUpload(FileUpload $upload, string $namespace, ?string $check
 
 		return new Image($this->friendly_url, $this->data_dir, $this->data_path, $identifier, [
 			'sha' => $checksum,
-			'name' => self::fixName($upload->getName()),
+			'name' => self::fixName($upload->getUntrustedName()),
```
Comment (paveljanda) [inline src/ImageStorage.php:127]: Could you please give me a hint why is this changed? Thanks.

Comment (petrparolek) [reply src/ImageStorage.php:127]: besause of deprecated of https://github.com/nette/http/blob/v3.1.5/src/Http/FileUpload.php#L67

Comment (paveljanda) [reply src/ImageStorage.php:127]: OK, nice, thank!

**Comment by paveljanda** [comment]
👍
